hugo.yaml

```
@@ -46,6 +46,11 @@ languages:
     languageCode: pt-BR
     params:
       description: Site do Projeto OpenTelemetry
+  uk:
+    languageName: Українська (Ukrainian)
+    languageCode: uk-UA
+    params:
+      description: Сайт проекту OpenTelemetry
```

```suggestion
      description: Сайт проєкту OpenTelemetry
```

Updated rules of Ukrainian spelling require writing the word project as "проєкт"